dialog/file.go

```diff
@@ -506,11 +517,15 @@ func (f *FileDialog) SetFilter(filter storage.FileFilter) {
 	}
 }
 
-// PresetFileName sets the filename in a FileSaveDialog.
+// SetFileName sets the filename in a FileSaveDialog.
```

Comment:
There is no type FileSaveDialog - so probably should be "FileDialog in save mode" or similar?